packages/headless/src/features/facets/category-facet-set/category-facet-set-actions.ts

```diff
@@ -21,6 +20,7 @@ import {
   NumberValue,
 } from '@coveo/bueno';
 import {validateCategoryFacetValue} from './category-facet-validate-payload';
+import {CategoryFacetValue} from '../../../controllers/facets/category-facet/headless-category-facet';
```

Comment:
I see this import is in a file in the `features` directory, coming from a file in the `controllers` directory.

I find this order a little strange, because it differs from how headless is structured. Going from the core to the edge, it's:

Store -> Reducers/Slices/Features -> Api -> Actions -> Controllers

I think it's tidier if the imports pull from a more core layer, rather than vice-versa?
Doesn't it feel weird that the reducer/action is getting the interface from the controller?